js/models/LocalSettings.js

```diff
@@ -16,7 +18,10 @@ export default class extends Model {
 
   defaults() {
     return {
-      macStyleWinControls: remote.process.platform === 'darwin',
+      windowControlStyle: remote.process.platform === 'darwin' ? 'mac': 'win',
+      showAdvancedVisualEffects: true,
+      saveTransactionMetadata: true,
+      defaultTransactionFee: 'high',
```

Comment:
Could we please get a validation for the defaultTransactionFee attribute. I'm assuming the valid values would be 'high', 'medium', 'low'?